tests/test_commodity_interfaces.py

```diff
@@ -2,100 +2,18 @@
 import numpy as np
 import matplotlib.pyplot as plt
 from matplotlib.patches import Patch
+from  data_adapter.preprocessing import Structure
 
-# Define filename to analyse
-name = 'sedos_set_io'
 
-# choose option
-# com_choice = 'secondary'
-com_choice = 'all'
 
-# Read the Excel file and process_set sheet
-df = pd.read_excel(name+'.xlsx', sheet_name='Process_Set')
+structure = Structure(
+    "SEDOS_Modellstruktur",
+    process_sheet="Processes_O1",
+    parameter_sheet="Parameter_O1",
+    helper_sheet="Helper_O1",
+)
 
-# Fill empty cells to facilitate logic - it is not considered as it has not one of the predefined category codes
-df.fillna('dummytext', inplace=True)
+structure.plot_commodity_interfaces()
 
-# Filter out processes which impair the logic
-df = df[~df['process'].str.contains('storage|transport|import|export')]
```

Comment:
In my last interface check I also filtered the following key words (due to changes in the model structure list):
storage|export|helper_source|helper_sink'

Reply:
As elaborated in henhuys comment my suggestion for the filters is as follows:

'storage|export||helper_sink|delivery'

As "_orig" could neglected I would also add the delivery processes.
What do you think?

Reply:
As I am not very deeply involved into discussions about naming and different edge cases, I don't really understand what you want to achieve with the filtering and what effect the exact process naming has. 
I don't understand what benefit comes not including sinks and sources in the plot?

I do understand that excluding `storage` makes sense to make the plot smaller. Even though it can have multiple outputs/inputs as  `helper_co2_storage_permanent` that has an output going into "nowhere" and for modelling I eventually need to be aware of such issues to help debugging.

Reply:
The idea of this plot first was mainly to check if there are defined inputs which are not being created or outputs which are not being used.

Therefore, temporal (storages) or spatial (delivery) shifts are disturbing this logic, as it might appear that e.g. electricity is present in our system as input and output, although there is no process to generate it.

For the sinks and exports, it is a modeling-specific decision I guess, and if it helps you, you can also keep them.

Reply:
Ah yes now I see, thank you!

So delivery should only be a kind of transport? As in the Model structure unfortunately there are processes that seem to be also doing Conversion. Such as `helper_co2_delivery`

Reply:
Yes, as we have a one-node model approach, the delivery is always modeled with a conversion between commodities.
For helper_co2_delivery different commodities are additionally merged.